clientv3/op.go

```diff
@@ -258,6 +264,10 @@ func withTop(target SortTarget, order SortOrder) []OpOption {
 	return []OpOption{WithPrefix(), WithSort(target, order), WithLimit(1)}
 }
 
+func WithGetDeletedKVs() OpOption {
```

Comment:
I do not like prevKV. deletedKV is more obvious than prevKV if we want the prefix.